v2/AdminCore.Common/Interfaces/IEventService.cs

```diff
@@ -7,7 +7,7 @@ namespace AdminCore.Common.Interfaces
 {
   public interface IEventService
   {
-    IList<EventDto> GetAnnualLeaveByEmployee(int employeeId);
+    IList<EventDto> GetEventByEmployee(int employeeId, EventTypes eventType);
```

Comment:
Now that you've changed this at a service level, is there not a corresponding change at the controller level?

Reply:
It's not currently being used by holiday or WFH controllers